distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -92,7 +92,7 @@ grant codeBase "file:/org.apache.felix.configadmin" {
     permission org.osgi.framework.BundlePermission "*", "provide, require, host, fragment";
 }
 
-grant codeBase "file:/org.apache.karaf.features.core/sync-installer-impl" {
+grant codeBase "file:/org.apache.karaf.features.core/sync-installer-impl/org.apache.karaf.features.command" {
```

Comment:
❓Does `org.apache.karaf.features.command` need all of these permissions or only the permission to read from `/etc`?

Reply:
It just needs permission for `/etc` but I wanted to reuse a grant block if possible due to the performance hit for every additional grant block. There was no grant block that granted permission just for `/etc`, and granting permissions to read from the `/examples` directory is not a security issue.

Reply:
There does seem to be some risk in allowing `org.apache.karaf.features.command` to _write_ to the /etc directory, though, right? Looks like that bundle contains all of the `feature:xxx` commands. It's probably fine to give it the same permissions as `org.apache.karaf.features.core`.